gradle.properties

```diff
@@ -55,6 +55,8 @@ checkstyle.version=10.19.0
 jdk8.checkstyle.version=9.3
 spotbugs.version=3.1.11
 
+# This only applies to the asm used in the tests. To override the asm used by the gradle plugins
+# change the version in the buildScript block of build.gradle.kts directly
```

Comment:
After using the `${property("asm.version")}` syntax this comment is no longer necessary.